server/socket.js

```diff
@@ -0,0 +1,106 @@
+import socketIO from'socket.io';
+import sticky from 'sticky-session';
+import http from 'http';
+import app from './server';
+import redisAdapter from 'socket.io-redis';
+import dotenv from 'dotenv';
+
+// Read .env file and set value in process.env
+dotenv.config()
+const DEFAULT_PORT = '8888'
+
+const listen = (expressApp, port = DEFAULT_PORT) => {
+  const io = socketIO();
+  const server = http.createServer(expressApp);
+
+  io.adapter(redisAdapter(process.env.REDIS_URL));
+  io.attach(server);
+  const isWorker = sticky.listen(server, port);
+  console.log(`localhost:${port}`);
+
+  if (isWorker) {
+    const game = io.of('/game').on('connection', (socket) => {
+      console.log(`connected, id: ${socket.id}`);
+
+      socket.on('getRooms', () => {
+        game.adapter.allRooms((err, rooms) => {
+          console.log(rooms); // an array containing all rooms (accross every node)
```

Comment:
When I start to write more in detail, I will remove console.log in this file. (This PR is for just settingup env.)